typings/fengari.d.ts

```diff
@@ -0,0 +1,186 @@
+  export const FENGARI_AUTHORS: string;
+  export const FENGARI_COPYRIGHT: string;
+  export const FENGARI_RELEASE: string;
+  export const FENGARI_VERSION: string;
+  export const FENGARI_VERSION_NUM: number;
+  export const FENGARI_VERSION_MAJOR: string;
+  export const FENGARI_VERSION_RELEASE: string;
+  export const FENGARI_VERSION_MINOR: string;
+
+  export type lua_String = Uint8Array;
+
+  export function to_luastring(s: string): lua_String;
+  export function to_jsstring(s: lua_String): string;
+  export function to_uristring(s: lua_String): string;
+
+  export namespace lua {
+    // tslint:disable:class-name
+    interface lua_State {
+    }
+    interface lua_Integer {
```

Comment:
Why is this an abstract interface?
`lua_Integer` is (usually) an integer type (subject to *conf.js)

Reply:
Because unfinished :). I wasn't sure about lua_Integer so just left it like that.